src/lib/y2partitioner/widgets/disk_description.rb

```diff
@@ -0,0 +1,65 @@
+require "cwm/widget"
+
+Yast.import "HTML"
+
+require "y2partitioner/widgets/blk_device_attributes"
+
+module Y2Partitioner
+  module Widgets
+    class DiskDescription < CWM::RichText
+      include Yast::I18n
+
+      def initialize(disk)
+        textdomain "storage"
+        @disk = disk
+      end
+
+      def init
+        self.value = disk_text
+      end
+
+    private
+
+      attr_reader :disk
+      alias_method :blk_device, :disk
+
+      include BlkDeviceAttributes
+
+      def disk_text
+        # TODO: consider using e.g. erb for this kind of output
```

Comment:
Yes, ERB is a good idea here I think. See 

- https://github.com/yast/yast-registration/blob/master/src/data/registration/certificate_summary.erb
- https://github.com/yast/yast-registration/blob/327ab34c020a89f8b7e3f4bff55deea82e457237/src/lib/registration/helpers.rb#L165

It would be nice to put these links to the comment so if anybody later decides to do the switch they will be helpful.